commentHandler.go

```diff
@@ -104,36 +104,72 @@ func findLabel(currentLabels []types.IssueLabel, cmdLabel string) bool {
 	return false
 }
 
+func classifyLabels(currentLabels []types.IssueLabel, labelAction string, labelValue string) ([]string, []string) {
+
+	var applicableLabels, unnecessaryLabels []string
+
+	labelValue = strings.Replace(labelValue, " ", "", -1)
```

Comment:
This needs changing.  What if the labelValue has contains a space, such as 'help wanted'?

Use TrimSpace in the range loop instead?